.github/workflows/integration_tests.yml

```diff
@@ -115,6 +116,15 @@ jobs:
           # Build and tag a docker container
           docker build --tag aleph-node:latest -f ./docker/Dockerfile .
 
+      - name: Generate chainspec
+        run: |
+          # NOTE : remove this step after keys are moved to a separate JSON file
```

Comment:
Perhaps moving such scripts into script files included in the image would make debugging process easier - you could do it inside of the image locally on your machine instead of checking it "remotely"?